tinker/e_announcements/__init__.py

```diff
@@ -28,10 +30,30 @@ def before_request(self, name, **kwargs):
         pass
 
     def index(self):
-        forms = self.base.traverse_xml(app.config['E_ANNOUNCEMENTS_XML_URL'], 'system-block')
+        username = session['username']
+
+        # Username is needed for caching purposes
+        @cache.memoize(timeout=600)
```

Comment:
we recently had some issues of these taking too long. can you reduce it to 5 min(300)?

Reply:
Done